apps/video/index.html

```diff
@@ -51,12 +51,12 @@
     <div id="app-container">
       <div id="two-column-spearator"></div>
       <section role="region" id="thumbnail-views">
-        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list">
+        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list" no-font-fit>
```

Comment:
I investigated the problem with the 'title-start' and 'title-end' by debugging the gaia-header code, so the font-fit logic was being applied at that time. It seems the code to remove the 'no-font-fit' attribute got lost after I did some refactoring.